source/css/sass/_mixins--spacing.scss

```diff
@@ -1,44 +1,156 @@
 @import "utility-functions";
-@import "variables--baseline-grid";
-@import "variables--breakpoints";
+@import "variables";
+
+@mixin _padding-left($size-in-px) {
```

Comment:
The `_` to indicate private? Sounds like the next version will solve this: https://github.com/sass/sass/issues/2470#issuecomment-372504651.

Reply:
If so, these should probably go at the bottom.

Reply:
Tried putting `_padding-right` and `_padding-left` mixins below `padding`, but reverted to prevent IDE having a hissy fit (don't want to get desensitised to errors).

Reply:
Might be worth commenting here (or somewhere) that things beginning with `_` are not part of the public contract in that case?

Reply:
Added a comment to that effect. It's in this file for now, but can be moved higher once extent of this approach is understood (there's another instance [`mixins--typography.scss`](https://github.com/libero/patterns-core/blob/master/source/css/sass/_mixins--typography.scss#L18)).